llama.cpp

```diff
@@ -7328,11 +7328,115 @@ void llama_sample_top_k(struct llama_context * ctx, llama_token_data_array * can
     }
 }
 
+void read_or_write_base_min_p(float* base_min_p) {
```

Comment:
I don't think you can really do it this way. You should use the existing sampler options setting stuff to pass in a filename to read/write stuff if necessary.

Reading/writing a file every time the sampler is called seems likely to be extremely inefficient.

Reply:
Mhm, as I describe in the post earlier this was useful for testing, but it not being a native option is very much so unmergeable